src/components/AeAccount.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <ae-card
+    :fill="fill"
+    class="ae-account">
+    <header>
+      <ae-identicon :address="address" />
+      <ae-input-plain
+        value="Main Account"
+        placeholder="Account name"
+        fill="white"
+      />
+      <div class="slot-icon">
+        <slot name="icon" />
+      </div>
+    </header>
+
+    <main>
+      <div class="security-status">{{ securityStatus }}</div>
+      <ae-address
+        :value="address"
+        length="medium"
+        gap="0" />
```

Comment:
```suggestion
        gap="0"
      />
```
Let's always more end of tag at a new line in accordance to trailing commas in JS to reduce [the diff size](https://eslint.org/docs/rules/comma-dangle).